wscript

```diff
@@ -968,16 +968,19 @@ def configure(ctx):
 
     if not ctx.dependency_satisfied('build-date'):
         ctx.env.CFLAGS += ['-DNO_BUILD_TIMESTAMPS']
-
+        
     if ctx.dependency_satisfied('clang-database'):
         ctx.load('clang_compilation_database')
-
+    
     if ctx.dependency_satisfied('cplugins'):
         # We need to export the libmpv symbols, since the mpv binary itself is
         # not linked against libmpv. The C plugin needs to be able to pick
         # up the libmpv symbols from the binary. We still restrict the set
         # of exported symbols via mpv.def.
-        ctx.env.LINKFLAGS += ['-rdynamic']
+        if ctx.dependency_satisfied('mingw'):
+            ctx.env.LINKFLAGS += ['-Wl,--out-implib,libmpv.dll.a']
```

Comment:
Same question here. (And why is it duplicated? I know it was duplicated before.)

Reply:
I replicated the behavior in that script (`-rdynamic` has been duplicated as well).

I intend to revert these changes anyway.
They're not needed if we build mpv.exe as a stub that links against libmpv.dll.